packages/cli/src/ErrorReporting.ts

```diff
@@ -41,10 +41,19 @@ export const initErrorHandling = async () => {
 	addGlobalEventProcessor((event, { originalException }) => {
 		if (originalException instanceof ExecutionBaseError && originalException.severity === 'warning')
```

Comment:
we should (in another PR), rename `ReportableError` to `N8NError` or `ApplicationError`, and then update all these other errors to extend from that instead. Then we can also replace `severity` with `level` everywhere, and also start adding node/credential types in `tags`, and workflowId/executionId in extras.

Reply:
Will do! Finally we have a basis to unify all this.